tamr_toolbox/notifications/common.py

```diff
@@ -0,0 +1,84 @@
+"""Tasks related to creation of Email notifications"""
+from typing import Union, List, Optional, Dict
+from tamr_unify_client import Client
+from tamr_unify_client.operation import Operation
+from tamr_toolbox.utils.operation import monitor
+
+from tamr_toolbox.models.operation_state import OperationState
+from tamr_toolbox.utils.operation import from_resource_id
+
+
+def monitor_job(
+    tamr: Client,
+    *,
+    send_message: object,
+    send_status_function: object,
```

Comment:
```suggestion
    send_message: Callable,
    send_status_function: Callable,
```
If these are functions, there is a specific `typing` object for that.  `Callable` defaults to `Callable[..., Any]` which means a function that takes any number of arguments of any type and returns any type of value.  Ideally, you would specify these but it may be hard in this case.

Reply:
Updated